site/PRD.md

```diff
@@ -0,0 +1,40 @@
+## NYC's Art Museum introduction
+
+New York has a large number of art galleries throughout the city, but many of them cannot meet the aesthetic requirements and travel expectations of visitors. Sometimes, certain selection needs to be made according to the characteristics and geographical characteristics of the art galleries.
+
+
+
+## data
+
+Open data from New York City and art gallery websites. At the same time, you need to locate the real-time location of the user, so you need map data.
+
+
+
+## Main display function
+
+The map shows the location of all the art galleries in New York City, all of which have the same map markings;
```

Comment:
It might be neat to put the building footprints on the map in addition to pointmarkers.